cfme/tests/infrastructure/test_host.py

```diff
@@ -312,3 +312,41 @@ def _finalize():
             actual_state, power_state
         ),
     )
+
+
+@test_requirements.general_ui
+def test_infrastructure_hosts_refresh_multi(
+    appliance, setup_provider, provider
+):
+    """
+    Polarion:
+        assignee: prichard
+        casecomponent: Infra
+        caseimportance: low
+        initialEstimate: 1/3h
+        testSteps:
+            1. Navigate to the Compute > Infrastructure > Hosts view.
+            2. Select at least 2 hosts (more if possible) by checking the box in upper left of
+               quadicons.
+            3. Click "Refresh Relationships and Power States" under the Configuration
+               dropdowm, and then click "OK" when prompted.
+        expectedResults:
+            1. Hosts view is displayed.
+            2.
+            3. "Refresh initiated for X Hosts from the CFME Database" is displayed in green
+               banner where "X" is the number of selected hosts. Properties for each host are
+               refreshed.
+    """
+    num_hosts = len(provider.data.get('hosts', {}))
+    if num_hosts < 2:
+        pytest.skip('not enough hosts to run test')
+    hosts_view = navigate_to(provider.collections.hosts, "All")
+    assert hosts_view.is_displayed
```

Comment:
I don't think it's necessary here, if the navigation is not successful, it will fail on `is_displayed` anyway